llvm/utils/TableGen/IntrinsicEmitter.cpp

```diff
@@ -346,7 +346,7 @@ static constexpr {} IIT_Table[] = {{
                 FixedEncodingTypeName);
 
   unsigned MaxOffset = 0;
-  for (auto [Idx, FixedEncoding, Int] : enumerate(FixedEncodings, Ints)) {
+  for (const auto [Idx, FixedEncoding, Int] : enumerate(FixedEncodings, Ints)) {
```

Comment:
I don't know why leaving `const` here. I had a quick search. `const auto [` is much smaller used in LLVM code base.
```
$ grep -r 'const auto \[' llvm/| wc
     37     275    3760
$ grep -r 'auto \[' llvm/| wc
    817    5871   82133
```

Reply:
Got it. It seems that we wont need to change anything in this PR. Closing this.